policy-service/src/main/java/uk/gov/gchq/palisade/service/policy/request/GetPolicyRequest.java

```diff
@@ -16,23 +16,21 @@
 
 package uk.gov.gchq.palisade.service.policy.request;
 
-
-import org.apache.commons.lang3.builder.EqualsBuilder;
-import org.apache.commons.lang3.builder.HashCodeBuilder;
-
 import uk.gov.gchq.palisade.Context;
 import uk.gov.gchq.palisade.ToStringBuilder;
 import uk.gov.gchq.palisade.User;
 import uk.gov.gchq.palisade.resource.LeafResource;
+import uk.gov.gchq.palisade.resource.Resource;
 import uk.gov.gchq.palisade.service.request.Request;
 
 import java.util.Collection;
+import java.util.Objects;
 
 import static java.util.Objects.requireNonNull;
 
 /**
  * This class is used in the request to get the policies that apply to the
- * collection of {@link LeafResource}'s.
+ * collection of {@link Resource}'s.
```

Comment:
Because the resource service will have already turned the requested resourceId into a collection of LeafResources, this should still be LeafResource
```suggestion
 * collection of {@link LeafResource}'s.
```